src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx

```diff
@@ -0,0 +1,174 @@
+    {
+        let aScore = MutationAssessorColumnFormatter.getData(a).score;
+        let bScore = MutationAssessorColumnFormatter.getData(b).score;
+        let aImpact = MutationAssessorColumnFormatter.getData(a).impact;
+        let bImpact = MutationAssessorColumnFormatter.getData(b).impact;
+
+        // use actual score values to compare (if exist)
+        if (aScore && bScore)
+        {
+            return aScore > bScore;
+        }
+        // if no score available sort by impact priority
+        else if (aImpact && bImpact)
+        {
+            let aPriority = -1;
+            let bPriority = -1;
+
+            if (MutationAssessorColumnFormatter.MA_SCORE_MAP[aImpact.toLowerCase()])
+            {
+                aPriority = MutationAssessorColumnFormatter.MA_SCORE_MAP[aImpact.toLowerCase()].priority;
+            }
+
+            if (MutationAssessorColumnFormatter.MA_SCORE_MAP[bImpact.toLowerCase()])
+            {
+                bPriority = MutationAssessorColumnFormatter.MA_SCORE_MAP[bImpact.toLowerCase()].priority;
+            }
+
+            return aPriority > bPriority;
+        }
+
+        return false;
+    }
+
+    public static filterValue(data:IColumnFormatterData):string
+    {
+        return MutationAssessorColumnFormatter.getDisplayValue(data);
+    }
+
+    /**
+     * Determines the display value by using the impact field.
+     *
+     * @param data  column formatter data
+     * @returns {string}    mutation assessor text value
+     */
+    public static getDisplayValue(data:IColumnFormatterData):string
+    {
+        let entry:any = MutationAssessorColumnFormatter.getMapEntry(data);
+
+        // first, try to find a mapped value
+        if (entry) {
+            return entry.label;
+        }
+        // if no mapped value, then return the text value as is
+        else {
+            return MutationAssessorColumnFormatter.getTextValue(data);
+        }
+    }
+
+    public static getTextValue(data:IColumnFormatterData):string
+    {
+        let maData:any = MutationAssessorColumnFormatter.getData(data);
+
+        // return impact value (if exists)
+        if (maData && maData.impact) {
+            return maData.impact;
+        }
+        else {
+            return "";
+        }
+    }
+
+    public static getScoreClass(data:IColumnFormatterData):string
+    {
+        let value:any = MutationAssessorColumnFormatter.getMapEntry(data);
+
+        if (value) {
+            return value.style;
+        }
+        else {
+            return "";
+        }
+    }
+
+    public static getMaClass(data:IColumnFormatterData):string
+    {
+        let value:any = MutationAssessorColumnFormatter.getMapEntry(data);
+
+        if (value) {
+            return "oma-link";
+        }
+        else {
+            return "";
+        }
+    }
+
+    public static getMapEntry(data:IColumnFormatterData):any // IMutationAssessorFormat?
+    {
+        let entry:any = null;
```

Comment:
There are very few situations where I would recommend using `any` type.  It's better to leave the type unspecified and let TypeScript figure it out through code flow analysis.